pkg/pillar/docs/zedmanager.md

```diff
@@ -0,0 +1,29 @@
+# Application Instance Management Agent in EVE (aka zedmanager)
+
+## Overview
+
+Zedmanager does the overall orchestration of application instances in EVE.
+This means receiving configuration from the controller via zedagent, and reporting results to the controller via zedagent, and then using other microservices to do their down:
+
```

Comment:
"to do their down" doesn't quite read for me -- did you mean "to do their own"?

Reply:
Fixed. Plus added some more text.

Reply:
Fixed